devicetypes/smartthings/testing/simulated-device-preferences.src/simulated-device-preferences.groovy

```diff
@@ -47,11 +47,16 @@ metadata {
 			input("enumInput3", "enum",
 					title: "Enum Title 3 (no options)", description: "Enum Description 3 (no options)",
 					required: false)
-			input("boolInput", "boolean",
+			input("booleanInput", "boolean",
 					title: "Boolean Title",
 					description: "Boolean Description",
 					defaultValue: "true",
 					required: false)
+			input("boolInput", "bool",
+					title: "Bool Title",
+					description: "Bool Description",
+					defaultValue: false,
```

Comment:
Did you intentionally have `bool` and `boolean` have different default values?

Reply:
Yes. I think it's beneficial to have both defaults for expanded test coverage.